packages/app/src/components/views/publication/components/PermissionSection.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { Grid, Typography, Button } from "@mui/material"
+import { palette, typography } from "../../../../theme"
+import AddIcon from "@mui/icons-material/Add"
+import { usePublicationContext } from "../../../../services/publications/contexts"
+import PermissionItem from "../../../commons/PermissionItem"
+import { useNavigate } from "react-router-dom"
+import { usersWithPermissions } from "../../../../utils/permission"
+
+export const PermissionSection: React.FC = () => {
+  const navigate = useNavigate()
+  const { publication, savePermission } = usePublicationContext()
+  const permissions = publication?.permissions || []
+  const usersPermissions = usersWithPermissions(permissions)
+  return (
+    <>
+      <Grid container justifyContent="space-between" alignItems={"center"} my={4}>
+        <Grid item>
+          <Typography
+            color={palette.grays[1000]}
+            variant="h5"
+            fontFamily={typography.fontFamilies.sans}
+            sx={{ margin: 0 }}
+          >
+            Permissions
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Button variant="contained" size="medium" onClick={() => navigate("/publication/permission/new")}>
+            <AddIcon style={{ marginRight: 13 }} />
+            New Permission
```

Comment:
We should probably not show this either when the user does not have the "Can edit the publication permissions" / publication/permissions

Reply:
With the other comment, this validation is solved. If the user doesn't have  "publication/permissions," they can't enter on this screen.

Reply:
@juliopavila: It was this button I menth:  😅  
<img width="163" alt="Screenshot 2022-04-13 at 14 49 41" src="https://user-images.githubusercontent.com/2156509/163183608-a21ff11b-a09b-420c-b89a-2f7d9eaf17df.png">

Reply:
They should probably still be able to see who has permissions (the list of addresses), just not the edit or add new button.

Reply:
You're right!